faf-commons-api/src/main/java/com/faforever/commons/api/dto/ModerationReport.java

```diff
@@ -0,0 +1,31 @@
+package com.faforever.commons.api.dto;
+
+import java.util.Collection;
+import java.util.Set;
+
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import com.github.jasminb.jsonapi.annotations.Type;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Type("moderationReport")
+@Getter
+@Setter
+public class ModerationReport extends AbstractEntity {
+  private String reportDescription;
+  private ModerationReportStatus reportStatus = ModerationReportStatus.AWAITING;
+  private String gameIncidentTimecode;
+  private String moderatorNotice;
+  private String moderatorPrivateNote;
+  @Relationship("bans")
+  private Collection<BanInfo> bans;
```

Comment:
BanInfo links ModerationReport and ModerationReport links BanInfo. Just saying, but you will notice soon enough you will need to make one relationShip manged and the other one not.